glossary.yml

```diff
@@ -6230,7 +6230,14 @@
       des objets du langage JavaScript. Contrairement à d'autres standards plus
       structurés comme [XML](#xml), JSON ne permet pas de définir un [schéma](#schema)
       de données, ni ne supporte l'ajout de commentaires.
-
+  uk:
+    term: "Нотація об'єктів JavaScript"
+    def: >
+      Спосіб представлення даних шляхом поєднання базових значень, таких як числа та рядки
+      символів, у структурах [списки](#list) та [ключ/значення](#dictionary). JSON
+      розшифровується як «JavaScript Object Notation»; на відміну від більш чітко визначених 
```

Comment:
Please do not use "fancy quotes" - use `"JavaScript Object Notation"` (you can prevent auto-conversion in your editor or operating system settings)